setup.py

```diff
@@ -37,6 +37,7 @@
         "keyring==18.0.1",
         "keyrings.alt==3.2.0",
         "py42>=1.7.0",
+        "texttable>=1.6.2",
```

Comment:
I would rather not take this on as a dependency. It is independently owned and hasn't been updated in almost a year -- as new versions of python continue to be released it's possible that this library will no longer work.

We already have functionality that we have written in this package for generically printing tables, see `format_to_table`.

Reply:
Removed.